remote_test.go

```diff
@@ -1633,6 +1634,42 @@ func testSwitchFrame(t *testing.T, c config) {
 	}
 }
 
+func testWait(t *testing.T, c config) {
+	newTitle := "Title changed."
+	titleChangeCondition:= func (wd WebDriver) (bool, error) {
```

Comment:
`gofmt -d remote_test.go` displays empty output, so I assume everything is okay, right?